driver-examples/src/main/java/com/datastax/driver/examples/json/JacksonJsonColumn.java

```diff
@@ -0,0 +1,170 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.examples.json;
+
+import com.datastax.driver.core.*;
+import com.datastax.driver.extras.codecs.json.JacksonJsonCodec;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.datastax.driver.core.querybuilder.QueryBuilder.*;
+
+/**
+ * Illustrates how to map a single table column of type {@code VARCHAR},
+ * containing JSON payloads, into a Java object using
+ * the <a href="http://wiki.fasterxml.com/JacksonHome">Jackson</a> library.
+ * <p/>
+ * This example makes usage of a custom {@link TypeCodec codec},
+ * {@link JacksonJsonCodec}, which is declared in the driver-extras module.
+ * If you plan to reuse this example, make sure to include the following
```

Comment:
I think it would be better to rephrase as 'If you plan to follow this example in your own project' instead of 'If you plan to reuse this example' or something to that extent.